system.properties

```diff
@@ -1 +0,0 @@
-java.runtime.version=1.8
```

Comment:
since this file is now empty, can we delete it? Is it mandatory to be present?

Reply:
it is deleted